model/daal4py/setup_common.py

```diff
@@ -0,0 +1,78 @@
+import os
+import sys
+import ast
+from pathlib import Path
+from setuptools import find_packages
+
+ORG = "intel"
+NAME = "dffml-model-daal4py"
+DESCRIPTION = "DFFML model daal4py"
+AUTHOR_NAME = "Ch.M.Hashim"
+AUTHOR_EMAIL = "hashimchaudry23@gmail.com"
+# Install dffml if it is not installed in development mode
+INSTALL_REQUIRES = ["daal4py>=0.2020.0"] + (
```

Comment:
Looks like we're missing `pandas` and `numpy` here

Reply:
> @pdxjohnny thanks for the detailed explanation. :)
> Although, I did set +e and -e as you suggested previously but it didn't work thats why I asked you if you made any other changes. Some weird behavior here.... I guess I should push the code so you can get a better idea.

Yes but I don't suppose it is the cause of the issue I'm facing locally because I still get the same log even after doing set +e and making these changes.